modules/azure/worker-as/worker-as.tf

```diff
@@ -1,20 +1,29 @@
-resource "azurerm_virtual_machine_scale_set" "workers" {
+resource "azurerm_linux_virtual_machine_scale_set" "workers" {
```

Comment:
azurerm_virtual_machine_scale_set is deprecated, azurerm_linux_virtual_machine_scale_set is the newer alternative and I need it for enabling termination events